plugins.xml

```diff
@@ -12669,4 +12669,142 @@ the registration of DOIs with mEDRA.</p>]]></description>
 			<description locale="pt_BR">Garante que a pergunta esteja relacionada ao revista atual.</description>
 		</release>
 	</plugin>
+	<plugin category="oaiMetadataFormats" product="biblat">
+		<name locale="es">Formato BIBLAT de Metadatos</nam
+		<name locale="en">BIBLAT Metadata Format</name>
+		<homepage>https://biblat.unam.mx/en/</homepage>
+
+		<summary locale="es">Provides a metadata structure consistent with the BIBLAT metadata format.</summary>
```

Comment:
El texto está intercambiado en inglés y español.
The English and Spanish text are swapped.

Reply:
Corregido